git_notes.md

```diff
@@ -0,0 +1,34 @@
+# Git Notes research
+
+The aim of this research is to provide
+a feedback around [git notes](https://git-scm.com/docs/git-notes) command.
+
+[Examples](./examples) directory contains [git_notes.sh](./examples/git_notes.sh).
+The command adds number of your specific commits into your `git` repository.
+
+## What git_notes.sh does?
+
+* Add N-notes into your `git repository`.
+  * The note added into git is:
+
+```Testing notes ${cnt}\npackit_test=yes"```
+* Show all notes added by previous command
+* Show specific (3th) note from git.
+
+## Can be used git notes as a database?
+
+`git notes` are basically one liners. We can use a separator and for
```

Comment:
can we store json in the note? could you please try to store some simple json object in a note and then deserialize it with python?